### PR TITLE
ICU-20693 Remember to mention the JAR setup

### PR DESCRIPTION
**Pull request by hagbard**
Added a sentence pointing people at the JAR file setup (this should have, obviously, always been there).

##### Checklist

- [X] Issue filed: https://unicode-org.atlassian.net/browse/ICU-20693
- [X] Updated PR title and link in previous line to include Issue number
- [X] Issue accepted
- [ ] Tests included
- [X] Documentation is changed or added

### CODE REVIEW

File: tools/cldr/cldr-to-icu/README.txt [hunk 1]
```diff
@@ -6,14 +6,34 @@
...
+
+* A CLDR release for supplying CLDR data and the CLDR API.
+* The Maven build tool
+* The Ant build tool
+
 Important directories
 ---------------------
 
-<CLDR_DIR>  = The root directory of the CLDR release.
+<CLDR_DIR>  = The root directory of the CLDR project.
```
Comment (gnrunge) [inline tools/cldr/cldr-to-icu/README.txt]: You may want to mention here or elsewhere that the CLDR directory in question should contain CLDR data in production form, not raw form.

Comment (sffc) [reply tools/cldr/cldr-to-icu/README.txt]: i.e., it should be cldr-staging.

Comment (hagbard) [reply tools/cldr/cldr-to-icu/README.txt]: Ahh yes, this was in flux as I was originally doing this and it slipped through the cracks.

**Comment by hagbard** [comment]
Goddamn github. How do I move just this commit from master to maint-66 without dragging all sorts of other stuff over????

**Comment by sffc** [comment]
> Goddamn github. How do I move just this commit from master to maint-66 without dragging all sorts of other stuff over????

On the command line:

```
$ git checkout hagbard-patch-1
$ git rebase --onto maint/maint-66 HEAD~1 hagbard-patch-1
$ git push -f
```

**Comment by sffc** [comment]
OK, we can put this on master.

**Pull request merged**